### PR TITLE
feat(suite): send form import tx from string

### LINKED ISSUE #5739: Import of transaction from csv format (PASTE)
With old wallet we could send funds to multiple addresses with "import" a csv with just a copy/paste .. a simple textarea.
With trezor suite, import is only possible if we select a local File with csv content.

A textarea to just paste the csv content (like in old wallet) would be great in trezor suite.
Copy/Paste is really nice on some situations.. otherwise we need to create a csv text file, paste csv content on file, save, upload to trezor suite and delete file. With paste into a textarea we just need to paste and go. Much much simple.

Maybe just turn trezor suite import compatible with file and paste in a textarea?

best,

### LINKED ISSUE #10066: Add input field to transaction import
**Design:** Let's discuss feasibility with @Hermez-cz please, we'll keep it simple together.

**Spec:**
When Sending a transaction, it can be imported from a CSV file which can be dropped into the modal window or selected from files.
Sometimes, users have the transaction copied in their paste bin, please add an input field for them to be able to paste the copied transaction in without having to create a file first.

Current implementation:
image

Deprecated Trezor Wallet solution for more context:
image

### PR DESCRIPTION
**Pull request by szymonlesisz**
## Description
While [fixing import tx](https://github.com/trezor/trezor-suite/pull/10327) i've noticed two other related issues

Here is working POC of import transaction from text, feel free to propose improvements to UI and copy

Screenshot from 2023-12-13 17-21-22
Screenshot from 2023-12-13 17-21-27

Resolve https://github.com/trezor/trezor-suite/issues/10066
Resolve https://github.com/trezor/trezor-suite/issues/5739

### CODE REVIEW

File: packages/suite/src/support/messages.ts [hunk 1]
```diff
@@ -1953,6 +1953,18 @@ export default defineMessages({
...
+        id: 'TR_IMPORT_CSV_FROM_FILE',
+    },
+    TR_IMPORT_CSV_FROM_TEXT: {
+        defaultMessage: 'Import as text',
+        id: 'TR_IMPORT_CSV_FROM_TEXT',
+    },
+    TR_IMPORT_CSV_FROM_TEXT_BUTTON: {
+        defaultMessage: 'Import',
+        id: 'TR_IMPORT_CSV_FROM_TEXT_BUTTON',
+    },
```
Comment (komret) [inline packages/suite/src/support/messages.ts]: There is the `IMPORT_CSV` key with the same value, let's use it to save our translators some work.

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/InputCSV.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-height: 250px;
+    align-items: center;
+`;
+
+const StyledTextArea = styled(Textarea)`
+    flex: 1;
+`;
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/InputCSV.tsx]: Same result, but a bit cleaner:

```suggestion
const Wrapper = styled.div`
    display: flex;
    flex-direction: column;
    min-height: 250px;
    justify-content: space-between;
    gap: 20px;
`;
```

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/TabSelection.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+    :hover {
+        border-bottom: 2px solid ${({ theme, selected }) => !selected && theme.STROKE_GREY};
+    }
+`;
+
+interface TabSelectionProps {
+    selectedTab: any;
+    setSelectedTab: any;
+}
```
Comment (komret) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/TabSelection.tsx]:
```suggestion
export type Tab = 'upload' | 'form';

interface TabSelectionProps {
    selectedTab: Tab;
    setSelectedTab: Dispatch<SetStateAction<Tab>>;
}
```

**Review by komret — CHANGES_REQUESTED** [review]
Great to finally have this enabled 👏 
Some nitpicks:

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/InputCSV.tsx (hunk 2 above)
Comment (szymonlesisz) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/InputCSV.tsx]: cool, done in [338cca7](https://github.com/trezor/trezor-suite/pull/10331/commits/338cca7524d208c082812c6d24c5b67e90e66944)

File: packages/suite/src/support/messages.ts (hunk 1 above)
Comment (szymonlesisz) [reply packages/suite/src/support/messages.ts]: done, [338cca7](https://github.com/trezor/trezor-suite/pull/10331/commits/338cca7524d208c082812c6d24c5b67e90e66944)

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/TabSelection.tsx (hunk 3 above)
Comment (szymonlesisz) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/TabSelection.tsx]: i forgot about types, thx, done [338cca7](https://github.com/trezor/trezor-suite/pull/10331/commits/338cca7524d208c082812c6d24c5b67e90e66944)

**Review by komret — APPROVED** [review]

**Pull request merged**